modules/ppcp-button/src/Assets/SmartButton.php

```diff
@@ -1082,6 +1081,23 @@ private function context(): string {
 		return $context;
 	}
 
+	/**
+	 * Checks if PayPal payment was already initiated (on the product or cart pages).
+	 *
+	 * @return bool
+	 */
+	private function is_paypal_continuation(): bool {
```

Comment:
Not sure if we use some other name for this state already, but if not, then "continuation" seems suitable.